src/headingengine.js

```diff
@@ -30,6 +23,39 @@ export default class HeadingEngine extends Plugin {
 	/**
 	 * @inheritDoc
 	 */
+	constructor( editor ) {
+		super( editor );
+
+		const t = editor.t;
+
+		/**
+		 * A set of default localized labels for `config.heading.formats`.
+		 *
+		 * @readonly
+		 * @protected
+		 * @member {Object} #_localizedFormatLabels
+		 */
+		const labels = this._localizedFormatLabels = {
+			'Paragraph': t( 'Paragraph' ),
+			'Heading 1': t( 'Heading 1' ),
+			'Heading 2': t( 'Heading 2' ),
+			'Heading 3': t( 'Heading 3' )
+		};
+
+		editor.config.define( 'heading', {
+			formats: [
+				{ id: 'paragraph', element: 'p', label: labels.Paragraph },
+				{ id: 'heading1', element: 'h2', label: labels[ 'Heading 1' ] },
```

Comment:
If the `_formats` getter localizes these labels, why are you using localized versions here too?